dashboard/src/components/AppView/AppView.tsx

```diff
@@ -292,6 +294,20 @@ export default function AppView() {
           <PageHeader
             title={releaseName}
             titleSize="md"
+            subtitle={
+              appDetails?.availablePackageRef ? (
+                <span>
+                  from package{" "}
+                  <Link
+                    to={url.app.packages.get(cluster, namespace, appDetails.availablePackageRef)}
+                  >
+                    {appDetails.availablePackageRef.identifier}
```

Comment:
Wondering why we're using the identifier here. I mean, right now it looks useful for the plugin shown, but the identifier used by a plugin may not be useful to users at all. We're getting the identifier here through the `appDetails`, which is an `AvailablePackageDetail`, so why not use `appDetails.displayName` ?

Maybe because that doesn't show the repository? I don't think that's a reason not to use it (and users can see that by clicking your new link if they want too), but not sure.

Anyway, I'll +1 but keen to understand why we're not using the display name of the available package ref, since that's what it's for.

Reply:
> Maybe because that doesn't show the repository? I don't think that's a reason not to use it (and users can see that by clicking your new link if they want too), but not sure.

Yep, that's precisely the reason why I used the identifier. Perhaps it is just me being so biased to seeing the `repo/foo` pattern, but perhaps: 1) the repo is not that useful for users; 2) if it happens to be, we can always added it to the display name.

So, 100% agree with the suggestion. Let's use the `displayName` to actually display the name of the package 😅

Reply:
![image](https://user-images.githubusercontent.com/11535726/169804974-55d6e806-330d-48c1-a555-24f28c4a4c05.png)

Reply:
Yep, exactly (RE we can update the display name if we want). Thanks Antonio.